docs/v1/integrations/openai.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: OpenAI
+description: "AgentOps provides first class support for OpenAI's GPT family of models"
+---
+
+import CodeTooltip from '/snippets/add-code-tooltip.mdx'
+import EnvTooltip from '/snippets/add-env-tooltip.mdx'
+
+<Note>
+This is a living integration. Should you need any added functionality, message us on [Discord](https://discord.gg/UgJyyxx7uc)!
+</Note>
+
+<Card title="OpenAI" icon="robot" href="https://www.openai.com">
+    First class support for GPT family of models
+</Card>
+
+<Steps>
+  <Step title="Install the AgentOps SDK">
+    <CodeGroup>
+      ```bash pip 
+      pip install agentops
+      ```
+      ```bash poetry
+      poetry add agentops
+      ```
+    </CodeGroup>
+  </Step>
+  <Step title="Install the OpenAI SDK">
+    <Note>
+      `openai<1.0.0` has limited support while `openai>=1.0.0` is continuously supported.
+    </Note>
+    <CodeGroup>
+      ```bash pip
+      pip install openai
+      ```
+      ```bash poetry
+      poetry add openai
+      ```
+    </CodeGroup>
+
+    To install `openai<1.0.0`, use the following:
+    <CodeGroup>
+      ```bash pip
+      pip install "openai<1.0.0"
+      ```
+      ```bash poetry
+      poetry add "openai<1.0.0"
+      ```
+    </CodeGroup>
+  </Step>
+  <Step title="Add 3 lines of code">
+    <CodeTooltip/>
+    <span className="api-key-container">
+    <CodeGroup>
+      ```python python
+      import agentops
+      from openai import OpenAI
+      
+      agentops.init(<INSERT YOUR API KEY HERE>)
+      client = OpenAI()
+      ...
+      # End of program (e.g. main.py)
+      agentops.end_session("Success") # Success|Fail|Indeterminate
+      ```
+    </CodeGroup>
+    </span>
+    <EnvTooltip />
+    <span className="api-key-container">
+      <CodeGroup>
+        ```python .env
+        AGENTOPS_API_KEY=<YOUR API KEY>
+        OPENAI_API_KEY=<YOUR OPENAI API KEY>
+        ```
+      </CodeGroup>
+      Read more about environment variables in [Advanced Configuration](/v1/usage/advanced-configuration)
+    </span>
+  </Step>
+  <Step title="Run your Agent">
+    Execute your program and visit [app.agentops.ai/drilldown](https://app.agentops.ai/drilldown) to observe your Agent! 🕵️
+    <Tip>
+      After your run, AgentOps prints a clickable url to console linking directly to your session in the Dashboard
+    </Tip>
+    <div/>
+    <Frame type="glass" caption="Clickable link to session">
+      <img height="200" src="https://github.com/AgentOps-AI/agentops/blob/cf67191f13e0e2a09446a61b7393e1810b3eee95/docs/images/link-to-session.gif?raw=true" />
```

Comment:
Can we kill the gif? Replace with a screenshot.

Also, instead of having the hash `.../blob/abc123...` make it point to `.../blob/main/`

Reply:
Nvm just did it